### ISSUE TITLE
Garbage not sent to opponent

### ISSUE THREAD

**Issue by bonifaadi** [o5k/tetrio-issues#1390]
### Checklist

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this issue is not already reported.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am using a **supported browser** or the Desktop Client.
- [X] I am not using anything that **modifies game code** (e.g. TETR.IO PLUS).
- [X] I am certain this is a **bug**, not a feature request.

### OS and Version

Windows 11

### Browser and version

Chrome (122.0.6261.70) tetr.io desktop 9.0.0 

### Describe the bug/issue

Possibly related to other issue reports regarding network and/or performance of the game but the network connection between two player was stable.
It happened one in my recent tetra league at the first game.

### Expected behavior

When the opponent clear the line (quad, etc) the garbage was not sent into my board. But, I could send the garbage to my opponent. In the entire first game, the opponent was only trying to survive because their garbage wasn't sent. I could clearly see him playing. I thought it was due to network connection but no garbage was sent at all for the entire first game even though it should still be sent even though there is a delay.

### Screenshots

https://tetr.io/#R:46cf1bfa09d0

https://github.com/user-attachments/assets/bd0b4d92-8fd5-4b62-ae16-33779ec3b396

### Additional context

I happened once and I don't know the cause.

**Comment by GoodOlDave** [comment]
can this be repeated?

**Comment by ZaptorZap** [comment]
No, nobody's found a reliable way to reproduce this, and there's no evidence of anyone using it maliciously.
